<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>i_main_container</class>
 <widget class="QDialog" name="i_main_container">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 56, 56);</string>
  </property>
  <widget class="QFrame" name="i_header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #252424</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="i_close">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>RB</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 8px;
	background-color: #38DBD0;
	color: #383838
}

QPushButton:hover {
	background-color: #23b2a8;
	color:  #383838
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="i_minmize">
    <property name="geometry">
     <rect>
      <x>1144</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>RB</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 8px;
	background-color: #38DBD0;
	color: #383838
}

QPushButton:hover {
	background-color: #23b2a8;
	color:  #383838
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="i_logout">
    <property name="geometry">
     <rect>
      <x>1118</x>
      <y>11</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/logout.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="i_right_menu">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>41</y>
     <width>891</width>
     <height>741</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 38, 38);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="i_choices">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="i_register_sec">
     <widget class="QLabel" name="i_register_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #38DBD0;</string>
      </property>
      <property name="text">
       <string>Register/Update Student</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="i_train_sec">
     <widget class="QLabel" name="i_train_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #38DBD0;</string>
      </property>
      <property name="text">
       <string>Train FR Model</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="i_offline_sec">
     <widget class="QLabel" name="i_offline_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #38DBD0;</string>
      </property>
      <property name="text">
       <string>Offline Attendance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="i_choose_video">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 12px;
	background-color: #38DBD0;
	color:rgb(255, 255, 255);

}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
      </property>
      <property name="text">
       <string>Choose Video</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="i_start">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 12px;
	background-color: #38DBD0;
	color:rgb(255, 255, 255);

}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="i_video_path">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #38DBD0;
border: none;
border-bottom: 2px solid rgb(85, 255, 255);
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="placeholderText">
       <string>Video Path...</string>
      </property>
     </widget>
     <widget class="QLabel" name="i_video_note">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ff0000</string>
      </property>
      <property name="text">
       <string>*There is no video specified</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="i_progress_bar">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>700</y>
        <width>761</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar{
	background-color:rgb(40, 120, 120);
	color: rgb(49, 147, 147);
	color: rgb(56, 56, 56);
	border-style:none;
	border-radius:10px;
	text-align:center;
}
QProgressBar::chunk{
	border-radius:10px; 
	background-color: qlineargradient(spread:pad, x1:0, y1:0.488591, x2:0.989474, y2:0.54, stop:0 #38DBD0, stop:1#38DBD0);
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="i_progress_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>693</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#38DBD0</string>
      </property>
      <property name="text">
       <string>Progress:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="i_settings_sec">
     <widget class="QLabel" name="i_settings_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #38DBD0;</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="i_left_menu">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>42</y>
     <width>301</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/Background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="i_logo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/SAQER1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="i_footer">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>787</y>
     <width>1201</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #38DBD0</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="i_register_student">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 25px;
	background-color: #38DBD0;
	color:rgb(255, 255, 255);

}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
   </property>
   <property name="text">
    <string>Register/Update Student</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/Register.png</normaloff>../images/Register.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="i_train_model">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 25px;
	background-color: #38DBD0;
	color: #ffffff
}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
   </property>
   <property name="text">
    <string>Train FR Model</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/Untitled-1.png</normaloff>../images/Untitled-1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="i_offline_atten">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 25px;
	background-color: #38DBD0;
	color:#ffffff
}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
   </property>
   <property name="text">
    <string>Offline Attendance</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/offliene.png</normaloff>../images/offliene.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="i_settings">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 25px;
	background-color: #38DBD0;
	color:#ffffff
}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/setting.png</normaloff>../images/setting.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
